### ISSUE TITLE
[bug report]Super long path cannot be displayed

### ISSUE THREAD

**Issue by Andy-AO** [pjeby/quick-explorer#26]
When the path is very long, it will simply not be displayed.

For example, `英语习得/《外语流利说：如何快速学习一门语言》怀纳/截面：怀纳的英语学习方案大概是什么样子的？如何从菜鸟到高手？/截面：怀纳的英语学习方案大概是什么样子的？如何从菜鸟到高手？`

![image](https://user-images.githubusercontent.com/38101170/146295167-085bc3e6-8df9-46e4-b4be-bde865773991.png)

Not showing is probably the worst option.

Omitting the beginning as `......`, or just truncating it is fine.

**Comment by pjeby** [comment]
Hm.  This can probably be fixed with CSS to change the wrapping or overflow and max-width of the items, but it's not immediately clear to me how to do that.  In the meantime, you can use the keyboard commands to open the current folder or root folder.

Also, a question: have you tried the Obsidian default theme?  Does it behave the same?  I know there are some themes that change stuff related to the titlebar that can affect QE's display (or hide it altogether).

**Comment by Andy-AO** [comment]
> Hm. This can probably be fixed with CSS to change the wrapping or overflow and max-width of the items, but it's not immediately clear to me how to do that. In the meantime, you can use the keyboard commands to open the current folder or root folder.
> 
> Also, a question: have you tried the Obsidian default theme? Does it behave the same? I know there are some themes that change stuff related to the titlebar that can affect QE's display (or hide it altogether).

I've been using the default theme.

**Comment by montemartin** [comment]
I'd like to suggest an alternate solution - move the entire plugin output from the title bar to the Status Bar. 

I was about to submit an issue to request that feature anyways, because on MacOS when the window is in fullscreen the title bar is hidden. Making the position a configurable option has the potential to solve several problems mentioned here without changing the default behavior.

**Comment by pjeby** [comment]
Yeah, I'm already leaning that way because 1) I originally wrote it for the statusbar and 2) there are an insane number of themes that dim or hide or otherwise mess with the title bar.  The downside of the status bar is that 1) cascading menus naturally from the bottom is **hard** (because eyes normally want to move down instead of up as we read) and 2) some of the themes also mess with the status bar, including in ways that would make the cascading problems *worse*.

That being said, this same issue is going to happen with the status bar unless there is CSS to fix it and/or I set it up to overflow by horizontal scrolling and then add code to make each bit scroll into view on demand.  (Not to mention possible needs for autoscrolling.)
